static/script-tests/tests/devices/mediaplayer/live/unseekable.js

```diff
@@ -84,13 +84,14 @@
     this.LivePlayerSupportLevelUnseekableTest.prototype.testSeekableLivePlayerRemoveAllEventCallbacksCallsFunctionInMediaPlayer = testFunctionsInLivePlayerCallMediaPlayerFunctions('removeAllEventCallbacks', 0);
```

Comment:
Should this have been renamed to 'playable', or is that a separate pull request? :-)